### PR TITLE
Added Kudos NFT for Token Script bounty

### PR DESCRIPTION
**Pull request by edsonayllon**
For this bounty: https://explorer.bounties.network/bounty/3257
- [Contract Address](https://etherscan.io/token/0x2aea4add166ebf38b63d09a75de1a7b94aa24163)
- [Contract  Actions](https://github.com/gitcoinco/Kudos721Contract/blob/master/contracts/Kudos.sol)
- [Address Holding Token](https://etherscan.io/address/0x614cc4a7388040b141aaefb9b2cff9e81c6c82ce#tokentxnsErc721)
- Token used in example [found here](https://gitcoin.co/kudos/2576/junior_release_manager)
`
20190815_015226000_iOS

20190815_015229000_iOS

### CODE REVIEW

File: examples/nft/kudos-example/token.en.shtml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            return ms
+        };
+
+        GeneralizedTime.prototype.getTimeZone = function () {
+            let time = this.rawData;
+            var length = time.length
+            var symbolIdx
+            if (time.charAt(length - 1 ) === 'Z'){
+                return 0
+            }
```
Comment (James-Sangalli) [inline examples/nft/kudos-example/token.en.shtml]: This is still being used?

Comment (edsonayllon) [reply examples/nft/kudos-example/token.en.shtml]: No. It didn't integrate well with the ERC721 contract. I'll work on removing it.

Comment (edsonayllon) [reply examples/nft/kudos-example/token.en.shtml]: 642e8c6

**Comment by edsonayllon** [comment]
The bounty this was for seems to have been deleted, so I'm closing this PR.

**Comment by edsonayllon** [comment]
Am going to delete this fork.

**Comment by James-Sangalli** [comment]
Hi @edsonayllon, the bounty has not been withdrawn, it was simply moved to the berlin bounty network. We are still holding this bounty and will announce the winner when the deadline ends (in 9 days) 

Please keep this PR up until the bounty has ended.

https://berlin.bounties.network/bounty/3257

**Comment by zhangzhongnan928** [comment]
> The bounty this was for seems to have been deleted, so I'm closing this PR.

@edsonayllon 
hi Mate, there must be some bugs with bounties.network😭. It seems they moved the Bounty to a special cat, here https://berlin.bounties.network/bounty/3257

It is still alive, and more bounties are coming 😁

**Comment by edsonayllon** [comment]
Apologies. Will re-initiate soon.

**Pull request closed without merging**